containers/Ideas/Phase.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {View} from 'react-native';
+
+import {TextSourceSans} from '../../components/TextSourceSans';
+import { styles } from './Phase.styles';
+
+export const Phase = props => {
+  const {activePhase, futurePhases, pastPhases} = props;
+  const startDate = (futurePhases) ? futurePhases[0].start_date : null;
```

Comment:
are the parentheses needed here and in the line below?

Reply:
I thought yes, but that was wrong :sweat_smile: So removed them now, thanks!